src/In.ProjectEKA.DefaultHip/LinkPatient.cs

```diff
@@ -52,39 +48,84 @@ public class LinkPatient : ILink
                     .Select(context => context.ReferenceNumber).ToArray()).ConfigureAwait(false);
 
             if (exception != null)
+            {
                 return new Tuple<PatientLinkReferenceResponse, ErrorResponse>
-                (null, new ErrorResponse(new Error(
-                    ErrorCode.ServerInternalError,
-                    "Unable to store data to Database")));
+                (null,
+                    new ErrorResponse(new Error(ErrorCode.ServerInternalError,
+                        ErrorMessage.DatabaseStorageError)));
+            }
 
-            await discoveryRequestRepository.Delete(request.TransactionId, request.Patient.ConsentManagerUserId)
-                .ConfigureAwait(false);
+            // await discoveryRequestRepository.Delete(request.TransactionId, request.Patient.ConsentManagerUserId)
+            //     .ConfigureAwait(false);
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Remove this commented out code.](https://app.codacy.com/gh/ProjectEKA/hip-service/pullRequest?prid=4807294)